project/project_repository.go

```diff
@@ -0,0 +1,184 @@
+package project
+
+import (
+	"log"
+
+	"golang.org/x/net/context"
+
+	"github.com/almighty/almighty-core/errors"
+	"github.com/almighty/almighty-core/gormsupport"
+	"github.com/jinzhu/gorm"
+	satoriuuid "github.com/satori/go.uuid"
+)
+
+// NewProjectRepository creates a new project repo
+func NewProjectRepository(db *gorm.DB) *GormProjectRepository {
+	return &GormProjectRepository{db}
+}
+
+// GormProjectRepository implements ProjectRepository using gorm
+type GormProjectRepository struct {
```

Comment:
When projects live in their own package, this can be renamed to `GormRepository`.